sql/gen-sql-config-docs.py

```diff
@@ -49,12 +50,13 @@ def generate_sql_configs_table(sql_configs, path):
 
     ```html
     <table class="table">
-    <tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
+    <tr><th>Property Name</th><th>Default</th><th>Meaning</th><th>Version</th></tr>
```

Comment:
shall we make it clear that this is since version?

Reply:
How about `Since Version`?

Reply:
SGTN

Reply:
OK